editions/tw5.com/tiddlers/filters/limit.tid

```diff
@@ -8,12 +8,15 @@ purpose: select the first or last <<.place n>> input titles
 
 <$macrocall $name=".operator-def"
 input="a [[selection of titles|Title Selection]]"
+suffix="optional, an offset"
+suffixName="o"
 parameter="an integer, defaulting to 0"
 paramName="n"
 output="the first <<.place n>> input titles"
-negationOutput="the last <<.place n>> input titles"
 />
 
-`limit[n]` and `!limit[n]` are synonyms for `first[n]` and `last[n]` respectively.
+The `limit[n]` filter returns at most the first <<.place n>> items from the input set. Using a negative  <<.place -n>> applies the limit from the end, thus returning the last  <<.place n>> elements.
+
```

Comment:
The terminology and phrasing here is inconsistent with the revised filter documentation, and the sentence "Using a negative …" effectively says the same thing twice. I suggest this wording:

`limit[n]` outputs the first <<.place n>> input titles. If <<.place n>> is negative, the <<.em last>> <<.place n>> titles are output instead. If the input contains less than <<.place n>> titles, the output is the same as the input.